Update monitor-data-plane-with-cockpit.mdx (#4366)
scaleway/docs-content: pages/kubernetes/how-to/monitor-data-plane-with-cockpit.mdx

1. Go to the **Cockpit** section of the Scaleway console, then click **Open dashboards**.
2. Log into Grafana using your [Cockpit credentials](/cockpit/how-to/retrieve-grafana-credentials/).

> 3. In Grafana's menu, go to dashboards and select **Kubernetes Cluster Pod Logs** to view logs collected from pods in your clusters.
> 4. **Filter pod logs** by:

Reviewer: ```suggestion
3. In Grafana's menu, navigate to **Dashboards** and select **Kubernetes Cluster Pod Logs** to view logs collected from pods in your clusters.  
4. **Filter pod logs** by:
```

   - `Datasource` which is automatically created upon deployment, and visible in the Cockpit console
   - `Cluster Name` ( e.g. `my-kapsule-cluster`)
   - `namespace`, `pod`, or `container` labels to isolate specific workloads
   - **Time range** to limit how far back in history you want to query
5. Alternatively, in Grafana's menu, navigate to **Dashboards** and select **Kubernetes Cluster Node Logs** to view system logs collected from nodes in your clusters.
6. **Filter node logs** by:
   - `Datasource` which is automatically created upon deployment and visible in the Cockpit console.

>    - `Cluster Name (e.g., `my-kapsule-cluster`).

Reviewer 2: ```suggestion
   - `Cluster Name` (e.g., `my-kapsule-cluster`).
```

> 5. Alternatively, in Grafana's menu, go to dashboards and select **Kubernetes Cluster Node Logs** to view system logs collected from nodes in your clusters.
> 6. **Filter node logs** by:
>    - `Datasource` which is automatically created upon deployment, and visible in the Cockpit console
>    - `Cluster Name` ( e.g. `my-kapsule-cluster`)
>    - `Node`, or `Syslog Identifier` labels to isolate specific workloads
>    - **Time range** to limit how far back in history you want to query
> 7. **Analyze** logs in real-time or historical mode to troubleshoot issues, watch for errors, or track performance.

Reviewer: ```suggestion
5. Alternatively, in Grafana's menu, navigate to **Dashboards** and select **Kubernetes Cluster Node Logs** to view system logs collected from nodes in your clusters.  
6. **Filter node logs** by:
   - `Datasource` which automatically created upon deployment and visible in the Cockpit console.  
   - `Cluster Name (e.g., `my-kapsule-cluster`).  
   - `Node` or `Syslog Identifier` labels to isolate specific workloads.  
   - `Time range` to limit how far back in history you want to query.  
8. **Analyze** logs in real-time or historical mode to troubleshoot issues, monitor errors, or track performance. 
```
